parse/analyze.go

```diff
@@ -136,8 +136,9 @@ type sqlContext struct {
 	ctes []*Relation
 	// outerScope is the scope of the outer query.
 	outerScope *sqlContext
-	// isAction is true if the visitor is analyzing SQL within an action.
-	isAction bool
+	// isInlineAction is true if the visitor is analyzing a SQL expression within an in-line
+	// statement in an action
+	isInlineAction bool
```

Comment:
Ah, so this was correctly false.  But where is it actually set in our Go code for in-line SQL? `scope()` just clones, but I don't see where it would actually be set to `true` anywhere.

Reply:
Whoops. Just fixed this, and added a test to confirm we receive the expected error when this occurs.